input/postgres/replication.go

```diff
@@ -76,12 +76,49 @@ SELECT client_addr,
 	FROM %s
  WHERE client_addr IS NOT NULL`
 
+const transactionIdSQLPg13 string = `
+SELECT
+	pg_current_xact_id(),
+	next_multixact_id
+FROM pg_catalog.pg_control_checkpoint()
```

Comment:
[note for reviewer]
The value of `pg_catalog.pg_control_checkpoint()` is only updated per checkpoint. The checkpoint runs every 5 mins (or wal size reaching to 1GB), so it's possible that it has pretty up-to-date value with full snapshot (runs every 10 mins). However, it is also common that the checkpoint timeout is set longer than 10 mins, and in that case, the `next_multixact_id` information can be stale.
As this value is more experimental one, we will accept this staleness and just treat it as "better than nothing".